rockcraft/models/project.py

```diff
@@ -326,6 +327,24 @@ def _validate_base_and_overlay(cls, item: Dict[str, Any], values) -> Dict[str, A
             )
         return item
 
+    @pydantic.validator("entrypoint_service", always=True)
```

Comment:
you shouldn't need the `always=True`, cause you don't want to do anything unless the field is actually set, which means you can also drop the `if entrypoint_service` below, cause the validator should only run when the field is set